assets/sass/06-components/header.scss

```diff
@@ -4,6 +4,11 @@
 	display: flex;
 	align-items: flex-start;
 	flex-wrap: wrap;
+	row-gap: var(--global--spacing-vertical); // Add space in case the menu wraps below the site branding.
```

Comment:
This should be tested on Safari... Though `row-gap` has great browser-support on grid layouts, Safari doesn't support it for flex layouts.

Reply:
There is already a screenshot from Safari above :)

Reply:
Ah you're right, I missed that :+1: